libs/blocks/merch-card/merch-card.css

```diff
@@ -0,0 +1,376 @@
+/* stylelint-disable selector-class-pattern */
+/* stylelint-disable-next-line selector-class-pattern */
+@media only screen and (min-width: 680px) {
```

Comment:
This seems weird to have a mobile only query. We should style mobile first w/ out a query and only change props on larger viewports.

Reply:
it's already defined and comes from caas